client/containers/Pub/PubDocument/PubDiscussions/Discussion/LabelSelect.tsx

```diff
@@ -116,7 +116,7 @@ class LabelSelect extends Component<Props, State> {
 												<Tooltip
 													content={
 														label.publicApply
-															? 'All discussion authors can apply this label.'
+															? 'All discussion contributors can apply this label.'
```

Comment:
Same as above, might do something like: `Anyone can apply this label to their discussions.`

Reply:
Changed to `All discussants`, as above.